Tests/Controller/Api/GalleryControllerTest.php

```diff
@@ -147,7 +147,7 @@ public function testPostGalleryMediaGalleryhasmediaAction()
         $view = $galleryController->postGalleryMediaGalleryhasmediaAction(1, 2, new Request());
 
         $this->assertInstanceOf('FOS\RestBundle\View\View', $view);
-        $this->assertSame(200, $view->getStatusCode(), 'Should return 200');
+        $this->assertSame(200, $view->getResponse()->getStatusCode(), 'Should return 200');
```

Comment:
The method is still there, but it doesn't call the response anymore.

Btw. I don't use the REST bundle. I simple want to fix the build ;)